index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+
+<body>
+    <header class="header">
+        <div class="logo-container">
+            <a href="#" class="my-logo">My Logo</a>
+        </div>
+        <nav class="menu">
+            <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
+        </nav>
+    </header>
+    <main class="main-container">
+        <section class="headline">
+            <h1 class="headline-title">I'm Misikir<br> Glad to see you!</h1>
+            <p class="headline-description">I’m a software developer! I can help you build a product , feature or
+                website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <a href="#">
+                <h2 class="connect-link">LET’S CONNECT</h2>
+            </a>
```

Comment:
- [ ] Good job adding the changes suggested by the previous reviewer, however, while doing them you underlined this text, kindly remove the line to stick to the Figma design(you can add text-decoration:none):

your design:
![links](https://user-images.githubusercontent.com/97834160/212030427-90815640-9e62-423a-ac51-36ce7698f9ca.png)

-[ OPTIONAL] Kindly make sure all the social icons are aligned properly to match the Figma design: